README.md

```diff
@@ -5,7 +5,4 @@
  `pip list`
 
 
- # DOCUMENTATION
-
- 
-
+ # DOCUMENTATION
```

Comment:
please complete the documentation file ASAP. and also add important cmds to the doc.